#Target Properties
int32 id
int32 state
int32 status


#Target Movement
float32 x
float32 y  

float32 vX
float32 vY

float32 radious 

#Timestamps are actually the number of the frame
int32 timestamp
